<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEditGroup</class>
 <widget class="QDialog" name="DialogEditGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>468</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edit Group</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="cat_common">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Common</string>
     </property>
     <layout class="QGridLayout" name="main">
      <item row="0" column="1">
       <widget class="QLineEdit" name="name"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="type">
        <item>
         <property name="text">
          <string>Basic</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Subscription</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="front_proxy_l">
        <property name="text">
         <string>Front Proxy</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="manually_column_width">
          <property name="text">
           <string>Manually column width</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="archive">
          <property name="text">
           <string>Archive</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="front_proxy">
          <property name="text">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="front_proxy_clear">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="cat_sub">
     <property name="title">
      <string>Subscription</string>
     </property>
     <layout class="QGridLayout" name="_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>URL</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="MyLineEdit" name="url"/>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="skip_auto_update">
        <property name="text">
         <string>Skip automatic update</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="cat_share">
     <property name="title">
      <string>Share</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="copy_links">
        <property name="text">
         <string>Copy profile share links</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="copy_links_nkr">
        <property name="text">
         <string>Copy profile share links (Neko Links)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>ui/widget/MyLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>name</tabstop>
  <tabstop>type</tabstop>
  <tabstop>front_proxy</tabstop>
  <tabstop>front_proxy_clear</tabstop>
  <tabstop>manually_column_width</tabstop>
  <tabstop>archive</tabstop>
  <tabstop>url</tabstop>
  <tabstop>skip_auto_update</tabstop>
  <tabstop>copy_links</tabstop>
  <tabstop>copy_links_nkr</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogEditGroup</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogEditGroup</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
